src/scss/06_components/paragraphs/_contact-card.scss

```diff
@@ -0,0 +1,78 @@
+.contact-card {
+  display: flex;
+}
+
+.contact-card--with-image {
+  @include breakpoint($breakpoint-s) {
+    flex-direction: column;
+  }
+}
+
+.contact-card--information-container {
+  @include breakpoint($breakpoint-s) {
+    max-width: 200px;
+  }
+}
+
+.contact-card--person-social-media {
+  display: flex;
+
+  .hel-icon {
+    height: 26px;
```

Comment:
26px sounds like a design bug, We should have things in multiples of 8.

Reply:
I'd recommend rounding to 24px and complaining to designer

Reply:
I'll change this!